.github/PULL_REQUEST_TEMPLATE/bep_template.yml

```
@@ -0,0 +1,29 @@
+---
+-   number:
+    title: # full title
+    display: # short title for display
+    google_doc:
+    pull_request: # pull request for the BEP on the BIDS specification repo
+    html_preview: # link to the html render of the pull request
+                  #  for the BEP on the BIDS specification repo
```

```suggestion
    title: # full title
    display: ' ' # short title for display
    google_doc: ' '
    pull_request: ' ' # pull request for the BEP on the BIDS specification repo
    html_preview: ' ' # link to the html render of the pull request
                     #  for the BEP on the BIDS specification repo
```

for harmonization with entries below?